sklearn/ensemble/gradient_boosting.py

```diff
@@ -88,12 +119,37 @@ def predict(self, X):
 class MeanEstimator(object):
     """An estimator predicting the mean of the training targets."""
     def fit(self, X, y, sample_weight=None):
+        """Fit the estimator.
+
+        Parameters
+        ----------
+        X : numpy array or sparse matrix of shape [n_samples,n_features]
```

Comment:
@NicolasHug can you try to be consistent with formatting of the docstrings.

It should be:

`X : {array-like, sparse matrix}, shape (n_samples, n_features)`

Reply:
Ok I will.
I didn't because the whole codebase is inconsistent already and I believe this problem would be better addresed globaly rather than localy (#11450, #11457).